build-info-extractor/src/main/java/org/jfrog/build/extractor/clientConfiguration/PrefixPropertyHandler.java

```diff
@@ -36,16 +38,25 @@ public String getPrefix() {
         return prefix;
     }
 
+    public String getDeprecatedPrefix() {
+        return ARTIFACTORY_PREFIX + prefix;
+    }
+
     public String getStringValue(String key) {
         return getStringValue(key, null);
     }
 
     public String getStringValue(String key, String def) {
-        String s = props.get(prefix + key);
-        if (s == null) {
-            s = def;
+        String value = props.get(prefix + key);
+        if (StringUtils.isNotBlank(value)) {
+            return value;
         }
-        return s;
+        value = props.get(ARTIFACTORY_PREFIX + prefix + key);
```

Comment:
Let's add a description here, explaining why we'er also using this prefix.
The same goes for the ```getBooleanValue``` and ```getIntegerValue``` methods in thsi class.